inlong-sort/sort-connectors/pulsar/pom.xml

```diff
@@ -54,6 +54,11 @@
             <groupId>org.apache.flink</groupId>
             <artifactId>flink-avro</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.inlong</groupId>
+            <artifactId>sort-connector-base</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
Add `<include>org.apache.inlong:*</include>`. Because now packaging will lost class file of connector-base.